src/test/scala/com/cognite/sdk/scala/v1/DataModelsTest.scala

```diff
@@ -174,23 +174,25 @@ class DataModelsTest extends CommonDataModelTestHelper with RetryWhile {
   }
 
   it should "work with ignore unknown externalIds" in initAndCleanUpData { _ =>
+    val unknownId = "toto"
     val outputIgnoreUnknownIds =
       blueFieldClient.dataModels
         .retrieveByExternalIds(
-          Seq("toto"),
+          Seq(unknownId),
           ignoreUnknownIds = true
         )
         .unsafeRunSync()
         .toList
     outputIgnoreUnknownIds.isEmpty shouldBe true
 
-    an[SdkException] should be thrownBy {
+    val error = the[CdpApiException] thrownBy {
```

Comment:
why does this change the error type?

Reply:
Because before they didn't return the missing ids error in the standard format:
```json
{
    "error": {
        "missing": [
            "Events"
        ],
        "code": 400,
        "message": "ids not found: Events"
    }
}
```
Now they fix it so we are able to decode the error message hehe 😁
```json
{
    "error":
    {
        "missing":
        [
            {
                "externalId": "Events"
            }
        ],
        "code": 400,
        "message": "ids not found: Events"
    }
}
```